build.gradle

```diff
@@ -214,6 +214,7 @@ dependencies {
 
     implementation 'com.squareup.okhttp3:okhttp'
     implementation 'org.springframework.boot:spring-boot-starter-actuator'
+    implementation 'org.springframework.hateoas:spring-hateoas'
```

Comment:
Just checking, this is being added because we want to move towards `hateoas`, correct? And this is a good PR to begin adding it?

Reply:
Yes, I think hateoas is a better way to write mvc tests than hardcoding URLs, and I do think this is a good PR to begin adding it, since this class didn't have any mockMvc tests yet.

On the other hand, using the hateoas APIs is a small part of the tests, and if we later decide we don't like this approach, it wouldn't be hard to remove it.